NETProvider/src/FirebirdSql.Data.UnitTests/TestsBase.cs

```diff
@@ -109,9 +116,10 @@ public virtual void TearDown()
 				}
 			}
 			connection.Close();
+			connection.Dispose();
```

Comment:
Looks like it doesn't help. I have tests that failing if run they like "Run all" but they don't fail if run them one by one.